pulsar-broker/src/main/java/com/yahoo/pulsar/broker/admin/BrokerStats.java

```diff
@@ -152,7 +152,7 @@ public LoadReport getLoadReport() throws Exception {
         // Ensure super user access only
         validateSuperUserAccess();
         try {
-            return ((SimpleLoadManagerImpl) pulsar().getLoadManager()).generateLoadReport();
+            return ((SimpleLoadManagerImpl) pulsar().getLoadManager().get()).generateLoadReport();
```

Comment:
Right, since `generateLoadReport` is part of `LoadManager`, I will not perform a cast here.